ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/SINGLE_LINK_TO_MLAG_PAIR.yml

```diff
@@ -3,8 +3,6 @@ fabric_name: SINGLE_LINK_TO_MLAG_PAIR
 
 type: l2leaf
 
-mlag_on_orphan_port_channel_downlink: false
```

Comment:
Please just flip this one to True instead of touching the other test. This feature was the whole purpose of this group.